js/site_nav.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+/* global require, module, document */
+
+function SiteNav(selector) {
+  this.$body = $(selector);
+  this.$dropdowns = this.$body.find('ul ul');
+  this.$toggles = this.$body.find('.js-toggle');
+  this.assignAria();
+  this.$toggles.change(this.onChange);
```

Comment:
`this.$toggles` is only used here, so may not need a variable assignment. We can also use event delegation to avoid binding a separate listener for each toggle:
```javascript
this.$body.on('change', '.js-toggle', this.onChange.bind(this));
```